cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1988,6 +1991,37 @@ pub(crate) fn define<'defs>(
         e.enc_32_64(bxor, rec_fa.opcodes(&PXOR));
     }
 
+    // SIMD bitcast from I32/I64 to the low bits of a vector (e.g. I64x2); this register movement
+    // allows SIMD shifts to be legalized more easily. TODO ideally this would be typed as an
```

Comment:
Per repo policy, if you would like to leave this TODO in the code for followup, please open an issue and reference the issue number in the comment.

Reply:
https://github.com/CraneStation/cranelift/issues/1146